docs/avdanos/contributing/design/OXYDE/secondary-style.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 3
+title: Window deployments
+description: Giving our app it's own layout.
+---
+
+:::caution
+Reminder: OXYDE stills in Beta phase.
+:::
+
+# Window deployments (aka App layouts)
+
+To make the experience consisten, AvdanOS has 5 different deploymentes (or layouts) for apps.
+![OXYDE Window deployments](Docsite/static/img/Window deployments.png)

```

Comment:
This is a big no-no:
Your file URLs should never have spaces in them. See #97 for more guidance on naming conventions.